src/app/Main/Body/QueryPage/Drawer/Filter/SelectorList-Selector.component.tsx

```diff
@@ -46,7 +46,7 @@ export function SelectorListSelectorComponent(
     const handleChange = (
         selectedOption: ValueType<{ value: string; label: string }>,
         keyName: FilterType | TableType
-    ): void => props.onChange(selectedOption, keyName);
+    ): void => props.onChange(selectedOption as { value: string; label: string }[], keyName);

```

Comment:
same here: 
```
if(selectedOption !== undefined && Array.isArray(selectedOption) {
  props.onChange(...)
} else {
  throw error or just do nothing or
  props.onChange([])
}
```

Reply:
Done. Commit "Type validation in selector functions"